PeerioTechnologies/peerio-desktop
# fix(chats): make scrolling to bottom instant when switching chats

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/1490/when-switching-chats-scrolling-to-bottom-shouldn-t-be-animated

#### Testing instructions  

Click on chats and ensure that autoscrolling to the latest message is not animated (that is, it's quite quick, but may be not exactly instant as more messages load). See GIF in clubhouse.

When loading previous unloaded "pages" of messages, though, autoscrolling should be smooth/animated.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
